sched/group/group_create.c

```diff
@@ -138,6 +139,9 @@ int group_allocate(FAR struct task_tcb_s *tcb, uint8_t ttype)
       return -ENOMEM;
     }
 
+  group->tg_info = (FAR struct task_info_s *)
+    kumm_zalloc(sizeof(struct task_info_s));
```

Comment:
is this supposed to work for CONFIG_BUILD_KERNEL?

Reply:
Look like we should use group_zalloc here